src/components/SavedRecipes.tsx

```diff
@@ -32,12 +65,14 @@ const SavedRecipes = () => {
   );
 
   const chooseCategory = (category: string) => {
-    const categorizedRecipes = recipes.filter((recipe: SavedRecipe)=> recipe.recipeCategory === category)
-    setFilteredRecipes(categorizedRecipes)
+    const categorizedRecipes = recipes.filter((recipe: SavedRecipe)=> recipe.recipeCategory === category);
+    setShow(true);
+    setFilteredRecipes(categorizedRecipes);
   };
```

Comment:
NON-BLOCKING: Do you want to be able to search and then filter the categories on that search or do you want to keep this as is where you can either filter by category or search?

Reply:
It is a nice idea! I haven't thought about that.

Reply:
Do you have any tips on how to do it better?

Reply:
Let's do this later (not in this PR) because it's going to require more complex logic and this does work as is. I'll outline some steps for how you can do that later today.

Reply:
Ok, thank you! Then I will merge this one and get back to this issue in the next PR.